userbenchmark/utils.py

```diff
@@ -68,15 +69,26 @@ def dump_output(bm_name, output, target_dir: Path=None) -> None:
 
 
 def get_date_from_metrics(metrics_file: str) -> str:
-    datetime_obj = datetime.strptime(metrics_file, "metrics-%Y%m%d%H%M%S")
-    return datetime.strftime(datetime_obj, "%Y-%m-%d")
-
+    if metrics_file.startswith("metrics-"):
+        datetime_obj = datetime.strptime(metrics_file, "metrics-%Y%m%d%H%M%S")
+        return datetime.strftime(datetime_obj, "%Y-%m-%d")
+    elif metrics_file.startswith("regression-"):
+        datetime_obj = datetime.strptime(metrics_file, "regression-%Y%m%d%H%M%S")
+        return datetime.strftime(datetime_obj, "%Y-%m-%d")
+    print(f"Unknown metrics or regression file name format: {metrics_file}")
+    exit(1)
```

Comment:
Right, but we could refactor like:
```suggestion
    assert metrics_file.startswith("metrics-") or metrics_file.startswith("regression-"), f"Unknown metrics or regression file name format: {metrics_file}"
    # metrics_file usually looks like metrics-%Y%m%d%H%M%S or regression-%Y%m%d%H%M%S
    stripped_filename = metrics_file.split("-")[1]
    datetime_obj = datetime.strptime(metrics_file.split("-")[1], "%Y%m%d%H%M%S")
    return datetime.strftime(datetime_obj, "%Y-%m-%d")
```

Reply:
Right, this would be cleaner. Thanks!